server/gorillamux/problems/docs/route-not-found.md

```diff
@@ -2,7 +2,16 @@
 
 # The requested endpoint could not be found
 
-The request was unable to be routed to one of the [specified](/v2/swagger) endpoints.
+The request was unable to be routed to one of the specified endpoints.
+
+## Fields
+
```

Comment:
We usually only include the custom fields, the others are described in the Index of the Problem documentation.

Reply:
OK